### DISCUSSION TITLE
What npm version is included in a distibution package?

### DISCUSSION THREAD

**Discussion by daemonofchaos** [nodesource/distributions#1812] [Q&A] [answered]
I am looking for information on what `npm` version is included in each distribution package. For example, in `node20` the `npm` version is `9.8.0`. Is this information listed anywhere?

**Comment by JesusPaz** [comment, marked as answer]
Hello, this [link ](https://stackoverflow.com/questions/51238643/which-versions-of-npm-came-with-which-versions-of-node)could help.

All the information can be found [here.](https://nodejs.org/dist/index.json)

**Reply by daemonofchaos** [reply]
Thank you!
